mcumgr-core/src/test/java/io/runtime/mcumgr/response/McuMgrResponseTest.java

```diff
@@ -0,0 +1,325 @@
+                 F4                             # primitive(20)
+                 69                             # text(9)
+                    636F6E6669726D6564          # "confirmed"
+                 F5                             # primitive(21)
+                 66                             # text(6)
+                    616374697665                # "active"
+                 F5                             # primitive(21)
+                 69                             # text(9)
+                    7065726D616E656E74          # "permanent"
+                 F4                             # primitive(20)
+                 FF                             # primitive(*)
+              BF                                # map(*)
+                 64                             # text(4)
+                    736C6F74                    # "slot"
+                 01                             # unsigned(1)
+                 67                             # text(7)
+                    76657273696F6E              # "version"
+                 65                             # text(5)
+                    332E302E30                  # "3.0.0"
+                 64                             # text(4)
+                    68617368                    # "hash"
+                 58 20                          # bytes(32)
+                    3AA147C63CD8E871104F7EBB1CD0B2E742D83DD79A581CB5AE0E05DB79444799 # ":\xA1G\xC6<\xD8\xE8q\x10O~\xBB\x1C\xD0\xB2\xE7B\xD8=\xD7\x9AX\x1C\xB5\xAE\x0E\x05\xDByDG\x99"
+                 68                             # text(8)
+                    626F6F7461626C65            # "bootable"
+                 F5                             # primitive(21)
+                 67                             # text(7)
+                    70656E64696E67              # "pending"
+                 F4                             # primitive(20)
+                 69                             # text(9)
+                    636F6E6669726D6564          # "confirmed"
+                 F4                             # primitive(20)
+                 66                             # text(6)
+                    616374697665                # "active"
+                 F4                             # primitive(20)
+                 69                             # text(9)
+                    7065726D616E656E74          # "permanent"
+                 F4                             # primitive(20)
+                 FF                             # primitive(*)
+              FF                                # primitive(*)
+           6B                                   # text(11)
+              73706C6974537461747573            # "splitStatus"
+           00                                   # unsigned(0)
+           FF                                   # primitive(*)
+
+         */
+        final byte[] data = { (byte) 0x01, (byte) 0x00, (byte) 0x00, (byte) 0xF4, (byte) 0x00, (byte) 0x01, (byte) 0x00, (byte) 0x00,
```

Comment:
If the data is the same all the time i suggest you to move this variable initialization to `@Before` function and init your test in `@Before` or `@BeforeAll` depending on case.

Reply:
No, data is different in each test.